### ISSUE TITLE
Accessibility violation in disabled toolbar

### ISSUE THREAD

**Issue by caritaou** [elyra-ai/canvas#2516]
ref 4063

![Image](https://github.com/user-attachments/assets/eea57f5a-2f29-4fe4-825a-18e077b42af1)
